src/main/java/com/radixdlt/crypto/ECDSASignature.java

```diff
@@ -102,6 +102,24 @@ public String toHexString() {
 		return Bytes.toHexString(getJsonR()) + Bytes.toHexString(getJsonS());
 	}
 
+	@Override
+	public boolean equals(Object o) {
+		if (this == o) {
+			return true;
+		}
+		if (o == null || getClass() != o.getClass()) {

```

Comment:
Unless there is a good reason for it, it's better to use the `instanceof` approach, rather than `getClass()`.

Reply:
Reverted back to `instanceof`, `getClass` was a remnant of... something. Most often relating to inheritance but this class was final so can't figure out what it was.